File: JAVA/PARCIAL II/EVA07-POO-TrujilloDavid/src/paquete1/Futbolista.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package paquete1;

/**
 *
 * @author Asus
 */
public class Futbolista extends Persona{
    protected double peso;
    protected String puesto;

    public Futbolista(int id, String nombre, int edad, double peso, String puesto) {
        super(id, nombre, edad);
        this.peso = peso;
        this.puesto = puesto;
    }
    
    public String JugarPartido(){
        return "El jugador "+super.nombre+" esta jugando un partido";
    }
    public String Entrenar(){
        return "El jugador "+super.nombre+" esta entrenando";
    }
    @Override
    public String Concentrarse() {
        return "El futbolista "+super.nombre+" se esta concentrando";
    }

    @Override
    public String Viajar() {
        return "El futbolista "+super.nombre+" esta viajando";
    }
    
    @Override
    public String toString(){
        return super.toString()+" Peso: "+peso+" Puesto: "+puesto;
    }
    
}

File: JAVA/PARCIAL II/EVA-POO-TrujilloDavid/EVA-POO-TrujilloDavid/Futbolista.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EVA_POO_TrujilloDavid
{
	internal class Futbolista : Persona, IFutbolista
	{
		private double peso;
		private string puesto;

		public Futbolista(int id, string nombre, int edad, double peso, string puesto) : base(id,nombre, edad)
		{
			this.peso = peso;
			this.puesto = puesto;
		}

		public string Puesto
		{
			get { return puesto; }
			set { puesto = value; }
		}
		public double Peso
		{
			get { return peso; }
			set { peso = value; }
		}

		public override string Concentrarse()
		{
			return "El futbolista "+base.Nombre+" se esta consentrando...";
		}

		public string Entrenar()
		{
			return "El futbolista " + base.Nombre + " esta entreanando...";
		}

		public string JugarPartido()
		{
			return "El futbolista " + base.Nombre + " esta jugando...";
		}

		public override string Viajar()
		{
			return "El futbolista " + base.Nombre + " esta viajan";
		}

		public override string ToString()
		{
			return base.ToString() + " Peso: " + peso + " Puesto: " + puesto;
		}
	}
}
